providers/aws/services/accessanalyzer/accessanalyzer_enabled_without_findings/accessanalyzer_enabled_without_findings_test.py

```diff
@@ -0,0 +1,164 @@
+from unittest import mock
+
+from providers.aws.services.accessanalyzer.accessanalyzer_service import Analyzer
+
+
+class Test_accessanalyzer_enabled_without_findings:
+    def test_no_analyzers(self):
+        accessanalyzer_client = mock.MagicMock
+        accessanalyzer_client.analyzers = []
+        with mock.patch(
+            "providers.aws.services.accessanalyzer.accessanalyzer_service.AccessAnalyzer",
+            new=accessanalyzer_client,
+        ):
+            # Test Check
+            from providers.aws.services.accessanalyzer.accessanalyzer_enabled_without_findings.accessanalyzer_enabled_without_findings import (
+                accessanalyzer_enabled_without_findings,
+            )
+
+            check = accessanalyzer_enabled_without_findings()
+            result = check.execute()
+
+            assert len(result) == 0
```

Comment:
Need to assert here that the result is `FAIL` when there is no access analyzers

Reply:
If there is no analyzers no results are generated.